DebugLogEnable.h

```diff
@@ -9,11 +9,11 @@
 #define LOG_SHORT_FILENAME (strrchr(__FILE__, '/') ? strrchr(__FILE__, '/') + 1 : strrchr(__FILE__, '\\') ? strrchr(__FILE__, '\\') + 1 : __FILE__)
 
 // C pre-proc Token Concatenation: https://wiki.sei.cmu.edu/confluence/display/c/PRE05-C.+Understand+macro+replacement+when+concatenating+tokens+or+performing+stringification
-#define xstr(s) str(s)
-#define str(s) #s
+#define MACRO_xstr(s) MACRO_str(s)
```

Comment:
If you don’t need this macro anymore, it’s good to `#undef MACRO_str` and `#undef MACRO_xtr`

Reply:
It may still be useful to users of the repo that may want to put strings into the LOG_PREAMBLE

Reply:
Oh wait this made me aware of another issue.... hold on

Reply:
Just changed the name of the macro so that it might be useful to developers down the road that would like to append string constants with macro calls

Reply:
Excellent. Could you also change README?

Reply:
README.md updated

Reply:
Thanks!